### PR TITLE
Updates swedish and italian translations

### PR DESCRIPTION
**Pull request by PureW**

### CODE REVIEW

File: locales/it-IT.json [hunk 1]
```diff
@@ -134,8 +134,8 @@
       "phrases": {
         "0": "Continua.",
         "1": "Continua su <STREET_NAMES>.",
-        "2": "Continue at <JUNCTION_NAME>.",
-        "3": "Continue toward <TOWARD_SIGN>."
+        "2": "Prosegui su <JUNCTION_NAME>.",
```
Comment (kevinkreiser) [inline locales/it-IT.json:137]: this and the other ones dont match previous translations, the lack of consistency could feel pretty random to a user, imho.

File: locales/it-IT.json [hunk 2]
```diff
@@ -410,9 +410,9 @@
       "phrases": {
         "0": "Entra nella rotonda.",
         "1": "Entra nella rotonda e prendi la <ORDINAL_VALUE> uscita.",
-        "2": "Enter the roundabout and take the <ORDINAL_VALUE> exit onto <ROUNDABOUT_EXIT_STREET_NAMES>.",
-        "3": "Enter the roundabout and take the <ORDINAL_VALUE> exit onto <ROUNDABOUT_EXIT_BEGIN_STREET_NAMES>. Continue on <ROUNDABOUT_EXIT_STREET_NAMES>.",
-        "4": "Enter the roundabout and take the <ORDINAL_VALUE> exit toward <TOWARD_SIGN>.",
+        "2": "Entra nella rotonda e prendi la <ORDINAL_VALUE> uscita per <ROUNDABOUT_EXIT_STREET_NAMES>.",
+        "3": "Entra nella rotonda e prendi la <ORDINAL_VALUE> uscita per <ROUNDABOUT_EXIT_BEGIN_STREET_NAMES>. Prosegui su <ROUNDABOUT_EXIT_STREET_NAMES>.",
+        "4": "Entra nella rotonda e prendi la <ORDINAL_VALUE> uscita verso <TOWARD_SIGN>.",
```
Comment (kevinkreiser) [inline locales/it-IT.json:415]: these ones match previous translations so that is nice!

File: locales/sv-SE.json [hunk 3]
```diff
@@ -38,8 +38,8 @@
         "1": "Håll <RELATIVE_DIRECTION> till <STREET_NAMES>.",
         "2": "Håll <RELATIVE_DIRECTION> till <BEGIN_STREET_NAMES>. Fortsätt på <STREET_NAMES>.",
         "3": "Håll <RELATIVE_DIRECTION> för att stanna kvar på <STREET_NAMES>.",
-        "4": "Bear <RELATIVE_DIRECTION> at <JUNCTION_NAME>.",
-        "5": "Bear <RELATIVE_DIRECTION> toward <TOWARD_SIGN>."
+        "4": "Ta <RELATIVE_DIRECTION> vid <JUNCTION_NAME>.",
+        "5": "Ta <RELATIVE_DIRECTION> mot <TOWARD_SIGN>."
```
Comment (kevinkreiser) [inline locales/sv-SE.json:42]: should the verbs really be different. to me it looks like it says:

"keep RELATIVE_DIRECTION..." in the old translation AND
"take a RELATIVE_DIRECTION..." in the new translation shouldn't we try to be consistent or do the different prepositions at the end of the phrase change the verb that sounds right with them?

File: locales/it-IT.json [hunk 4]
```diff
@@ -2137,13 +2137,13 @@
...
         "un mezzo miglio",
-        "a quarter mile",
+        "un quarto di miglio",
         "<FEET> piedi",
         "meno di 10 piedi"
       ],
       "example_phrases": {
         "0": [
-          "In a quarter mile, Turn right onto North Gay Street."
+          "In un quarto di miglio, Turn right onto North Gay Street."
```
Comment (danpaz) [inline locales/it-IT.json:2146]: You can revert this change, this is an example phrase.

File: locales/sv-SE.json [hunk 5]
```diff
@@ -2137,15 +2137,15 @@
...
         "en halv mile",
-        "a quarter mile",
+        "fyra hundra meter",
         "<FEET> fot",
         "mindre än 10 fot"
       ],
       "example_phrases": {
         "0": [
-          "In a quarter mile, Turn right onto North Gay Street."
+          "In fyra hundra meter, Turn right onto North Gay Street."
```
Comment (danpaz) [inline locales/sv-SE.json:2146]: Here too, revert example phrase.

File: locales/it-IT.json (hunk 1 above)
Comment (mandeepsandhu) [reply locales/it-IT.json:137]: I have this weird feeling that they've simply copy-pasted the english instructions into an online translation service, and put the results back here! :raised_eyebrow:

**Comment by PureW** [comment]
Closing PR since new translations are different from existing translations within the same group.
